src/test/groovy/com/blackducksoftware/integration/hub/detect/bomtool/nuget/NugetInspectorPackagerTest.groovy

```diff
@@ -8,6 +9,7 @@ import com.blackducksoftware.integration.hub.detect.nameversion.NameVersionNodeT
 import com.google.gson.Gson
 import com.google.gson.GsonBuilder
 
```

Comment:
Why are we ignoring this test?

Reply:
Jordan's new version of the inspector broke all of his tests. They need to be rewritten. I don't think this change should be in this branch though.